concepts/observability-strategy/strategy.md

```diff
@@ -1,54 +1,60 @@
+  - Reporting is done via a Grafana installation which loads pre-bundled dashboards; dashboards can be added at runtime
+- Traces
+  - 
 
-Observability can be split up into:
-1. Instrumentation of the users application
-2. The collection and preprocessing of the signals from the user application and the surrounding infrastructure, including metadata enrichment
-3. The delivery of the signal to an backend for storage
-4. An optional aggregation of the signals
-5. Storage of the signals
-6. The analysis, querying, and dashboarding of the signals
+The current solution provides a feature-rich end-to-end setup at a first glance. However, at a second look users realize soon the major drawbacks and usually will require additional stacks deployed besides:
+- Very limited integration possibilities into external systems; that is usually required for different reasons (cross-cluster correlation, forensic analysis, long time storage,..); integration points are not vendor-neutral
+- Very limited configuration options for data enrichment/filtering; users are interested in specific aspects for specific scenarios
+- Storage backends are non-scalable and with can be used only in limited scenarios. There is no option to upgrade the backend into a scalable setup but also no way to integrate with other solutions
 
-![a](./assets/stages.drawio.svg)
+### Requirements
 
-Kyma is a runtime to operate actual workloads. To fulfill typical operational requirements, users must be able to observe the workload deployed to the runtime .
+Integration (and with that, moving the focus away from in-cluster backends) is the key to open up the stack for a broad range of use cases. Users can bring their own backends if they already use a commercial offering or run their own infrastructure. The data can be stored outside the cluster in a managed offering, shared with the data of multiple clusters, away from any tampering or deletion attempt of a hacker, to name just a few advantages.
 
-### Mandatory feature set
+Providing ready-to-use in-cluster backends is necessarily opinionated, does not cover all usage scenarios, and does not fit into Kyma's goal of providing the Kubernetes building blocks to integrate into the the SAP ecosystem. Also, the licensing issues (particularly with Grafana and Loki, or ElasticSearch as an alternative backend technology) show that an opinionated stack is problematic. It's better to handle opinionation by integrating with actual managed services. 
 
-The mandatory parts for a Kyma runtime are the stages that must be happen inside the runtime. With that, the following points are the major goal of the Kyma feature set:
+In order to support integration as a key element, it is mandatory that a Kyma runtime supports the stages that must happen within a runtime itself (instrumentation, collection, delivery). Thus, the following points will be the major goal of the Kyma feature set:
```

Comment:
Why don't you like my headline suggestion?

```suggestion
### Mandatory features

To support integration as a key element, a Kyma runtime must support the stages that happen within the runtime itself (instrumentation, collection, delivery). Thus, the following points are mandatory for the future Kyma observability stack:
```
Trying to simplify that further.